src/incremental/compareCIL.ml

```diff
@@ -23,15 +23,17 @@ type change_info = {
   mutable unchanged: global list;
   mutable removed: global list;
   mutable added: global list;
-  mutable force_reanalyze: VarinfoSet.t;
+  mutable exclude_from_rel_destab: VarinfoSet.t;
```

Comment:
Why is `force_reanalyze` removed here, but in `change_status` there's still a case for it?

Reply:
The set `force_reanalyze` was up to now only used to exclude the contained functions from reluctant destabilization. Since functions with a changed header should be handled exactly the same way, I collect both in the same set and renamed it to `force_reanalyze` such that it is clear what functions it contains.

In change_status I kept them separate such that it is more obvious for which reason a function is added to the `force_reanalyze` set. One could probably also get rid of this distinction since they are handled the same way and it produces some duplicate code. Its just that, in my opinion, a `changedFunHeader` and a `ForceReanalyze` function do not really share a property that could be considered a `change_status`.